Plugin/LSP/LSPOutlineView.cpp

```diff
@@ -0,0 +1,1160 @@
+#include "LSPOutlineView.h"
+
+#include "LSP/UI.h"
+#include "LSP/LSPManager.hpp"
+#include "LSPSymbolStyle.h"
+#include "ColoursAndFontsManager.h"
+#include "clAnsiEscapeCodeColourBuilder.hpp"
+#include "globals.h"
+#include "imanager.h"
+#include "macros.h"
+
+wxDEFINE_EVENT(wxEVT_LSP_OUTLINE_SYMBOL_ACTIVATED, wxCommandEvent);
+wxDEFINE_EVENT(wxEVT_LSP_OUTLINE_VIEW_DISMISS, wxCommandEvent);
+
+namespace {
+    class wxIntClientData : public wxClientData {
+        int m_value = 0;
+    public: 
+        wxIntClientData(int value) : wxClientData(), m_value(value) {}
+        ~wxIntClientData() = default;
+        
+        int GetValue() const { return m_value; }
+        void SetValue(int value) { m_value = value; }
+    };
+}
+
+LSPOutlineView::LSPOutlineView(wxWindow* parent, wxWindowID id, long style)
+    : wxPanel(parent, id), m_style(style)
+{
+    // initialize behaviour from config
+    m_initialiseAllCollapsed = clConfig::Get().Read(kConfigOutlineStartAllCollapsed, false);
+    m_initialiseClassesCollapsed = clConfig::Get().Read(kConfigOutlineStartClassesCollapsed, false);
+    m_initialiseSubclassesCollapsed = clConfig::Get().Read(kConfigOutlineStartSubclassesCollapsed, true);
+    m_initialiseEnumsCollapsed = clConfig::Get().Read(kConfigOutlineStartEnumsCollapsed, true);
+    m_initialiseFunctionsCollapsed = clConfig::Get().Read(kConfigOutlineStartFunctionsCollapsed, true);
+    m_initialiseContainersCollapsed = clConfig::Get().Read(kConfigOutlineStartContainersCollapsed, false);
+    m_initialiseVariablesCollapsed = clConfig::Get().Read(kConfigOutlineStartVariablesCollapsed, true);
+    m_keepNamespacesExpanded = clConfig::Get().Read(kConfigOutlineKeepNamespacesExpanded, true);
+    m_showDetails = clConfig::Get().Read(kConfigOutlineShowSymbolDetails, true);
+    m_showSymbolKind = clConfig::Get().Read(kConfigOutlineShowSymbolKinds, false);
+    m_forceTree = clConfig::Get().Read(kConfigOutlineForceTree, true);
+    m_sortType = (LSPSymbolParser::SortType) clConfig::Get().Read(kConfigOutlineSortType, (int)LSPSymbolParser::SortType::SORT_LINE);
+    
+    // load bitmaps
+    auto images = clGetManager()->GetStdIcons();
+    m_bitmaps.reserve((int)LSPSymbolStyle::BITMAP_MAX);    
+    // order has to be same as LSPSymbolStyle::BITMAP enum
+    m_bitmaps.push_back(images->LoadBitmap("symbol-module"));
```

Comment:
The UI should be consistent. Please use the symbols that are already being used by the code completion box. If you are not happy with the way they look - you can submit a later PR to change them